mslib/idp/idp_conf.py

```diff
@@ -207,13 +203,13 @@ def sso_dir_path(local_file):
 
 # Authentication contexts
 
-    #(r'verify?(.*)$', do_verify),
+# (r'verify?(.*)$', do_verify),
 
 CAS_SERVER = "https://cas.umu.se"
```

Comment:
The site does not exists, maybe a user wants to have another address there. Can this become a config var? For what is it used?